src/tests/ftest/aggregation/dfuse_space_check.py

```diff
@@ -1,9 +1,8 @@
 """
-  (C) Copyright 2020-2024 Intel Corporation.
+  Copyright 2025 Hewlett Packard Enterprise Development LP
```

Comment:
Both copyrights need to remain. In total
```suggestion
  (C) Copyright 2020-2024 Intel Corporation.
  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
```

Reply:
Done

Reply:
This still isn't right. Please see my comment above

Reply:
Sorry I missed the `(C)` in front.